src/VirtualClient/VirtualClient.Actions/BlenderBenchmark/BlenderBenchmarkExecutor.cs

```diff
@@ -0,0 +1,269 @@
+                            this.CaptureMetrics(process, commandArguments, relatedContext, process.StandardOutput.ToString(), scene);
+                        }
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Returns true/false whether the component is supported on the current
+        /// OS platform and CPU architecture.
+        /// </summary>
+        protected override bool IsSupported()
+        {
+            bool isSupported = base.IsSupported()
+                && this.Platform == PlatformID.Win32NT
+                && this.CpuArchitecture == Architecture.X64;
+
+            if (!isSupported)
+            {
+                this.Logger.LogNotSupported("BlenderBenchmark", this.Platform, this.CpuArchitecture, EventContext.Persisted());
+            }
+
+            return isSupported;
+        }
+
+        /// <summary>
+        /// Processes benchmark results
+        /// </summary>
+        private void CaptureMetrics(IProcessProxy workloadProcess, string commandArguments, EventContext telemetryContext, string resultsContent, string scenario)
+        {
+            if (workloadProcess.ExitCode == 0)
+            {
+                try
+                {
+                    if (resultsContent == null)
+                    {
+                        throw new WorkloadResultsException(
+                            "No results output from Blender Benchmark",
+                            ErrorReason.WorkloadResultsNotFound);
+                    }
+
+                    BlenderBenchmarkMetricsParser resultsParser = new BlenderBenchmarkMetricsParser(resultsContent);
+                    var metrics = resultsParser.Parse();
+
+                    this.MetadataContract.AddForScenario(
+                           scenario,
+                           workloadProcess.FullCommand());
+                    this.MetadataContract.Apply(telemetryContext);
+
+                    this.Logger.LogMetrics(
+                        "BlenderBenchmark",
+                        scenario,
+                        workloadProcess.StartTime,
+                        workloadProcess.ExitTime,
+                        metrics,
+                        null,
+                        commandArguments,
+                        this.Tags,
+                        telemetryContext,
+                        toolVersion: this.Package.Version);
+                }
+                catch (SchemaException exc)
+                {
+                    EventContext relatedContext = telemetryContext.Clone()
+                        .AddError(exc);
+
+                    this.Logger.LogMessage($"{nameof(BlenderBenchmarkExecutor)}.WorkloadOutputParsingFailed", LogLevel.Warning, relatedContext);
+                }
+            }
+        }
+
+        private async Task InitializeBlenderBenchmarkAsync(EventContext telemetryContext, CancellationToken cancellationToken)
```

Comment:
Added blender engine and scene installation as part of the init package function.